site/package.json

```diff
@@ -41,6 +41,7 @@
         "eslint-plugin-react-hooks": "^4.6.0",
         "eslint-plugin-vitest": "^0.0.48",
         "jsdom": "^21.1.0",
+        "picocolors": "^1.0.0",
```

Comment:
What's this needed for?

Reply:
to run npm run test it was causing an error potentially something vitest needs

Reply:
removed tested it again and it works without